client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -2015,11 +2023,13 @@
                 });
 
                 metadata.initialize().then(function() {
-                    scope.item.hasCrops = false;
                     scope.metadata = metadata.values;
-                    scope.item.hasCrops = scope.metadata.crop_sizes.some(function (crop) {
-                        return scope.item.renditions[crop.name];
-                    });
+                    if (scope.item.type === 'picture'){
+                        scope.item.hasCrops = false;
```

Comment:
I added this flag, just for images, to check if images are cropped, to know to change label on button, so this flag is not needed outside if, because items of different type cannot have crops, only images can have it.